tests/safety/test_volkswagen_mqb.py

```diff
@@ -225,7 +225,10 @@ def test_rx_hook(self):
       if msg == MSG_MOTOR_20:
         to_push = self._gas_msg(0)
       self.assertTrue(self._rx(to_push))
-      to_push[0].RDHR ^= 0xFF
```

Comment:
This isn't equivalent, should be only on the LSB, no?
```
to_push[0].data[4] ^= 0xFF
```

Reply:
yes, it's a bug, fixed